src/components/flyout/flyout.tsx

```diff
@@ -301,10 +290,19 @@ export const EuiFlyout = forwardRef(
         closeButtonProps?.className
       );
 
+      const closeButtonStyles = euiFlyoutCloseButtonStyles(euiTheme);
+
+      const closeButtonCSSStyles = [
+        closeButtonStyles.euiFlyout__closeButton,
+        closeButtonStyles[`closeButton--${closeButtonPosition}`],
+        side === 'left' && closeButtonStyles['closeButton--outside-left'],
```

Comment:
see previous comment - we've generally kept `__` BEM naming in our Emotion classNames, but we've removed `--` modifiers in favor of basic camelCased vars since Emotion concatenates classNames for us

```suggestion
        closeButtonStyles.euiFlyout__closeButton,
        closeButtonStyles[closeButtonPosition],
        side === 'left' && closeButtonStyles.outsideLeft,
```